gammapy/modeling/models/cube.py

```diff
@@ -642,13 +644,21 @@ class FoVBackgroundModel(ModelBase):
 
     def __init__(
         self,
+        dataset_name,
         spectral_model=None,
-        dataset_name=None,
         spatial_model=None,
         covariance_data=None,
     ):
-        if dataset_name is None:
-            raise ValueError("Dataset name a is required argument")
+        # TODO: remove this in v2.0
+        if isinstance(dataset_name, SpectralModel):
+            warnings.warn(
+                "dataset_name has been made a positional argument since v2.0.",
```

Comment:
I think you are right, I got confused with the wording "since"...

Positional arguments are arguments that need to be given in the right order (this the proper name for *args), complementary to keyword arguements.

Reply:
So, it is "until"?